development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
```

Comment:
Right.  Thankfully that specific use case (replacing a column definition with a different one in a plugin) is explicitly supported through the ``@Session.model_mixin`` API.